change/@fluentui-api-docs-2021-02-11-15-12-52-removeCardFromWebsite.json

```
@@ -1,8 +1,7 @@
 {
-  "type": "patch",
+  "type": "prerelease",
   "comment": "Removing react-cards from api-docs generation.",
   "packageName": "@fluentui/api-docs",
   "email": "humbertomakotomorimoto@gmail.com",
-  "dependentChangeType": "patch",
-  "date": "2021-02-11T23:12:52.474Z"
+  "dependentChangeType": "patch"
```

Not sure what happened here
```suggestion
  "dependentChangeType": "patch",
  "date": "2021-02-11T23:12:52.474Z"
```

---

Ummm appears that a really recent beachball version stopped putting dates in the files...../

---

Oh yeah, not sure either.